.nvmrc

```diff
@@ -1 +1 @@
-6
```

Comment:
I feel like there was discussion of this change, but I can't recall. Can you?

Reply:
Yup it's because `async` wasn't available so you recommended updating node here